js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,20 @@
+const isError = str => typeof str !== 'string';
+
+function abbreviateString(inputString) {
+// error check
+  if (isError(inputString)) throw new TypeError('Input should be string');
+
+  if (inputString === ' ') return inputString;
```

Comment:
What if there are two spaces? You can test for whitespaces or trim the string and then check the length.